README.md

```diff
@@ -71,16 +71,16 @@ Or *jsDelivr*:
 
 ### Loading the editor
 
-After installing and/or importing the module, you can load EasyMDE onto the first TextArea on the web page:
+After installing and/or importing the module, you can load EasyMDE onto the first `textarea` element on the web page:
 
-```html
+```html`
```

Comment:
The extra ` breaks the code rendering.

```suggestion
```html
```

Reply:
My bad. Fixed.